mslib/msui/topview.py

```diff
@@ -191,6 +191,8 @@ def __init__(self, parent=None, mainwindow=None, model=None, _id=None,
         """
         Set up user interface, connect signal/slots.
         """
+        if parent is None:
+            parent = mainwindow
```

Comment:
This looks like undoing the changes for
https://github.com/Open-MSS/MSS/issues/2007

Likly the fix was not good, but it needs to stay solved

Reply:
See: https://github.com/Open-MSS/MSS/pull/2100#discussion_r1441545102